pkg/build/api/v1/types.go

```diff
@@ -541,6 +541,9 @@ type BuildLogOptions struct {
 	// is not available yet. Otherwise the server will wait until the build has started.
 	// TODO: Fix the tag to 'noWait' in v2
 	NoWait bool `json:"nowait,omitempty" description:"if true indicates that the server should not wait for a log to be available before returning; defaults to false"`
+
+	// Version of the build for which to view logs.
+	Version *int `json:"version,omitempty" description:"the version of the build for which to view logs"`
```

Comment:
Should this be consistent with `DeploymentLogOptions.Version`? https://github.com/openshift/origin/blob/d92321dc3bea3090a415d8889044a1997669d0c5/pkg/deploy/api/types.go#L379